### ISSUE TITLE
Can realizables be realized in stases?

### ISSUE THREAD

**Issue by eliasweatherfield** [CommonCoreOntology/CommonCoreOntologies#107]
From the definition of BFO:realizes:

"Paraphrase of elucidation: a relation between a process and a realizable entity, where there is some material entity that is bearer of the realizable entity and participates in the process, and the realizable entity comes to be realized in the course of the process"

It seems to me that stases (CCO:Stasis) do not have a "course" in which a realizable could be realized.

I would appreciate any comments.

**Comment by rorudn** [comment]
I don't think so either. It's one thing to state that a screwdriver's function of delivering torque is realized in the turning of a screw and another to state that the screwdriver was in an undamaged state (bore it's functions). It is true that the processes which realize functions occur during the same temporal interval of the undamaged state.

**Comment by APCox** [comment]
My initial thought is that it seems acceptable to say that a stasis has part a process that realizes some, e.g., function. But it is not OK to say that the stasis itself realizes that function.

**Comment by eliasweatherfield** [comment]
Interesting. I always assumed stasis could only have stasis as parts, but maybe I got process parthood wrong all along.

At least the definition of CCO:has process part does not seem to preclude it:

"x has_process_part y iff x and y are instances of Process, such that y occurs during the temporal interval of x, and y either provides an input to x or receives an output of x, or both."

@APCox May I ask, would you also allow Intentional Acts as subprocesses of Natural Processes? This is issue arose in #90

**Comment by cameronmore** [comment]
This issue, along with others having to do with stases, should be moved into the discussion section of the repo.
